habitat-baselines/habitat_baselines/config/default_structured_configs.py

```diff
@@ -347,6 +347,14 @@ class DDPPOConfig(HabitatBaselinesBaseConfig):
 @attr.s(auto_attribs=True, slots=True)
 class TrainerAgentConfig(HabitatBaselinesBaseConfig):
     name: str = "SingleAgentAccessMgr"
+    ###############################
+    # Population play configuration
```

Comment:
What is the default here? SingleAgentAccessMgr seems to indicate there is only one agent in the AgentAccessMgr, but the attributes bellow seems to indicate population based training is enabled.

Reply:
I will make the default 1 to make this more clear. It is only enabled if the AgentAccessMgr type is properly set.